src/functionaltest/java/com/ericsson/ei/config/SeleniumConfig.java

```diff
@@ -83,24 +75,32 @@ public static String getBaseUrl(int randomServerPort) {
         return "http://localhost:" + randomServerPort;
     }
 
+    private static FirefoxBinary installFirefoxBinary() {
+        String[] firefoxBZip2FileUrlLinuxBits = firefoxBZip2FileUrlLinux.split("/");
+        String firefoxBZip2FileNameLinux = firefoxBZip2FileUrlLinuxBits[firefoxBZip2FileUrlLinuxBits.length-1];
+
```

Comment:
I think you can use FilenameUtils.getName() function for getting the filename from the Url.
Check this page:
https://commons.apache.org/proper/commons-io/javadocs/api-2.5/org/apache/commons/io/FilenameUtils.html

Reply:
Oh thats seems better, i'll use that thanks :)